nixos/doc/manual/development/meta-attributes.section.md

```diff
@@ -38,3 +39,12 @@ file.
     ```ShellSession
     $ nix-build nixos/release.nix -A manual.x86_64-linux
     ```
+
+-  `buildDocsInSandbox` determines whether the option documentation
+   for the module can be built in a derivation sandbox, and thus cached
+   between builds. The default is `true` and should usually not be
+   changed; set it to `false` only if the module requires access to `pkgs`
+   in its documentation (e.g. because it loads information from a linked
+   package to build an option type) or if its documentation depends on
+   other modules that also aren't sandboxed (e.g. by using types defined
+   in the other module).
```

Comment:
Please explain why the sandbox is used and why it imposes restrictions on option attributes.